### ISSUE TITLE
Not pulling some packages

### ISSUE THREAD

**Issue by oliverdain** [ionos-cloud/reprepro#21]
I have a reprepro setup like so:

```
# conf/distributions 
Origin: Mendel
Label: Mendel
Codename: eagle
Architectures: arm64
Components: main
Description: Mendel Linux Mirror
Update: - mendel
Log: logfile
```

```
# conf/updates
Name: mendel
Method: https://mendel-linux.org/apt/eagle
FilterList: hold list
VerifyRelease: blindtrust
```

And my list file contains lots of things including:

```
libavfilter7:arm64				install
libavformat-dev:arm64				install
libavformat58:arm64				install

...

libusb-1.0-0:arm64                              install
libusb-1.0-0-dev:arm64                          install
```

When I run `reprepro -b . --noskipold -v --show-percent update eagle` I see it pull lots and lots of other packages specified in my `conf/list` file but for some reason it completely skips `libusb-1.0-0` and `libavfilter`, `libavformat`, etc. In fact, if you look at `pool` the `liba` and `libu` directories are entirely missing:

```
$ ls pool/main/
a  b  c  d  e  f  g  h  i  j  k  l  libc  libd  libe  libh  libi  libj  libl  libs  libt  libu  libw  libx  m  n  o  p  r  s  t  u  v  w  x  z
```

I don't get any error messages and the files _are_ available in the source repo (e.g. `curl http://mendel-linux.org/apt/eagle/pool/main/libu/libusb-1.0/libusb-1.0-0_1.0.22-2_arm64.deb` works as expected).

Any ideas?

**Comment by oliverdain** [comment]
Note: the `libavfilter`, `libavformat`, etc. files are actually here: http://mendel-linux.org/apt/eagle/pool/main/f/ffmpeg/ but are not mirrored to the corresponding location in my `pool/` directory:

```
$ ls pool/main/f/ffmpeg/
ffmpeg_4.1.6-1~deb10u1_arm64.deb
```

**Comment by oliverdain** [comment]
Sorry, also relevant:

```
$ reprepro --version
reprepro: This is reprepro version 5.3.0
```

On Debian Bullseye.

**Comment by bdrung** [comment]
I have currently not much time to debug this. Can you bisect the commit where this problem starts?

**Comment by oliverdain** [comment]
Thanks for the reply @bdrung . I'm afraid in also in the middle of a time crunch. Any chance you could suggest a version you think is more likely to work?

**Comment by bdrung** [comment]
No, since I don't use this feature. If you can come up with a small test case (that does not require Internet access), I could try to bisect it. The tests live in `tests/basic.sh` and `tests/multiversion.sh`.
